tracker/tracker.go

```diff
@@ -49,12 +49,25 @@ func (f *FilterConfig) buildHash() {
 	for _, i := range f.Address {
 		h.Write([]byte(i.String()))
 	}
-	for _, i := range f.Topics {
-		if i == nil {
+	for _, topics := range f.Topics {
+		if topics == nil {
 			h.Write([]byte("empty"))
-		} else {
-			h.Write([]byte(i.String()))
+
+			continue
+		}
+
+		innerTopicArray := make([]byte, 0)
```

Comment:
Why do you create another array here instead of piping everything into Hash?

Reply:
You're right 👍 

Since it's just aggregating, no need to have this level of control.

Fixed 👍 

c4cfe2e